mw-webapp/src/component/wayCollectionCard/WayCollectionCard.tsx

```diff
@@ -41,12 +49,64 @@ interface WayCollectionProps {
    * Data attribute for cypress testing
    */
   dataCy?: string;
+
+  /**
+   * If a card editable and removable
+   */
+  isEditable?: boolean;
+
+  /**
+   * Callback triggered on title edit
+   */
+  onTitleEdit?: (name: string) => void;
+
+  /**
+   * Callback triggered on collection delete
+   */
+  onDelete?: () => void;
 }
 
 /**
  * WayCollectionCard component
  */
 export const WayCollectionCard = (props: WayCollectionProps) => {
+  const [isRenameCollectionModalOpen, setIsRenameCollectionModalOpen] = useState(false);
+  const title = (
+    <Title
+      level={HeadingLevel.h3}
+      text={props.collectionTitle}
+      className={styles.title}
+      placeholder=""
+    />);
+  const editModal = (
+    <Modal
+      isOpen={isRenameCollectionModalOpen}
```

Comment:
I moved "title" because it's used twice in the code.
Both "editModal" and "deleteConfirm" moved to a variable for readability. They clutter the component

Reply:
Заголовок всегда отображается. Давай немного изменим логику и заголовок будем рендерить всегда ,а вот бургер еню только если props.isEditable. Тогда title не будет повторяться в коде)

Reply:
То есть я имею в виду, давай переместим проверку isEditable с 118 на 121 строку

Reply:
Done